src/common/cli_helper.h

```diff
@@ -91,7 +91,11 @@ static void ProcessPaths(
   }
 }
 
-// helper to convert a Yaml node recursively into a string
+// helper to convert a YAML node recursively into a string
+//
+// TODO: create a helper function that converts a YAML node into a string
+// without an emitter; consider extracting YAML-related helper functions to a
```

Comment:
I am somewhat sure that I have written that function. Did you accidentally delete it?

Reply:
What's bad about the emitter?